configure.ac

```diff
@@ -637,7 +637,7 @@ fi
 # This option sets the $with_spidermonkey variable.
 AC_ARG_WITH([spidermonkey],
 	[AS_HELP_STRING([--with-spidermonkey],
-		[disable SpiderMonkey Mozilla JavaScript engine support])])
+		[enable ECMAScript browser scripting (requires SpiderMonkey)])])
 
```

Comment:
browser  scripting in ELinks means something else. See guile, lua, perl, python, ruby hooks for what it is.
I agree that disable -> enable is good change here, but the rest is still misleading.

Reply:
Ok. Would something like the following would suffice?

```
--with-spidermonkey     enable SpiderMonkey Mozilla Javascript engine
--disable-sm-scripting  disable ECMAScript browser scripting
```